### PR TITLE
Fixed list.scrollTop not returning initialScrollTop value (fixes #80)

### LINKED ISSUE #80: list.scrollTop returns -1 when using `initialScrollTop` option
``` js
var list = new FastList({
  initialScrollTop: 50,
  ...
});

// wait till rendered ...

list.scrollTop; //=> -1;
list.els.container.scrollTop; //=> 50
```

### PR DESCRIPTION
**Pull request by wilsonpage**

### CODE REVIEW

File: test/data-source.js [hunk 1]
```diff
@@ -92,7 +92,7 @@
     },
 
     getFullHeight: function() {
-      this.data.length * itemHeight;
+      return this.data.length * itemHeight;
```
Comment (wilsonpage) [inline test/data-source.js:95]: Can't believe we missed this, ooopsie! :stuck_out_tongue:

**Comment by wilsonpage** [comment]
@etiennesegonzac r?

File: test/fast-list-test.js [hunk 2]
```diff
@@ -563,4 +563,58 @@ suite('FastList >', function() {
...
+    });
+
+    test('it sets the scrollTop of the container before render()', function() {
+      source.initialScrollTop = 100;
+      var fastList = new FastList(source);
+
+      // trigger mutation callback
+      scheduler.mutation.yield();
+
+      assert.isTrue(setScrollTop.calledBefore(fastList.render));
```
Comment (etiennesegonzac) [inline test/fast-list-test.js:598]: nit: also an `calledWith` assert?

**Comment by etiennesegonzac** [comment]
r=me!

**Pull request merged**